javascript/templates/api/api.mustache

```diff
@@ -2,6 +2,7 @@
 import { BaseAPIRequestFactory, RequiredError } from './baseapi{{extensionForDeno}}';
 import {Configuration} from '../configuration{{extensionForDeno}}';
 import { RequestContext, HttpMethod, ResponseContext, HttpFile} from '../http/http{{extensionForDeno}}';
+import { randomBytes } from 'crypto';
```

Comment:
This is incorrect, you don't have `crypto` on the web, please use `Math.random()` or another more cross-platform method.